job_vars/os-brick-ceph.yaml

```diff
@@ -0,0 +1,296 @@
+vms:
+  - devstack:
+    name: "dv-osbrick-{{ zuul_change }}-{{ zuul_patchset }}-{{ job_type }}"
+    image: bionic-09.08.2019
+    flavor: devstack
+    inventory_group: devstack
+    additional_params: 'ansible_ssh_private_key_file=/home/jenkins-slave/.ssh/id_rsa.pem ansible_ssh_common_args="-o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null"'
+    userdata: ""
+    tags: "dv-{{ zuul_project | basename }}-{{ zuul_change }}-{{ zuul_patchset }}-{{ job_type }}"
+  - hv2016-compute:
+    name: "hv-osbrick-{{ zuul_change }}-{{ zuul_patchset }}-{{ job_type }}"
+    image: win2016-hypervrole
+    flavor: ci-win2016hv
+    inventory_group: win2016-compute
+    additional_params: "ansible_winrm_cert_pem=/home/jenkins-slave/ssl/ssl_winrm.crt ansible_winrm_cert_key_pem=/home/jenkins-slave/ssl/ssl_winrm.key ansible_winrm_transport=certificate ansible_ssh_user=administrator ansible_connection=winrm ansible_winrm_server_cert_validation=ignore ansible_ssh_port=5986"
+    userdata: "{{ lookup('file', '/home/jenkins-slave/ssl/ssl_winrm.crt') }}"
+    tags: "hv-{{ zuul_project | basename }}-{{ zuul_change }}-{{ zuul_patchset }}-{{ job_type }}"
+
+win_crt: /home/jenkins-slave/ssl/ssl_winrm.crt
+win_key: /home/jenkins-slave/ssl/ssl_winrm.key
+log_key: /home/jenkins-slave/.ssh/id_rsa_log.pem
+log_path: /tmp/build_logs/{{ zuul_change }}-{{ zuul_patchset }}/{{ job_type }}
+
+## devstack
+
+#cherry_picks:
+#  - project: openstack/cinder
+#    path: "{{ devstack_dir.stack }}/cinder"
+#    patches:
+#      - refs/changes/41/403641/4
+#      - refs/changes/19/426719/4
+#  - project: openstack/os-win
+#    path: "{{ devstack_dir.stack }}/os-win"
+#    patches:
+#      - refs/changes/41/403641/4
+#      - refs/changes/19/426719/4
+
+local_conf:
+  enable_services:
+    - rabbit
+    - mysql
+    - key
+    - n-api
+    - n-crt
+    - n-cond
+    - n-sch
+    - n-cauth
+    - neutron
+    - q-svc
+    - q-agt
+    - q-dhcp
+    - q-l3
+    - q-meta
+    - q-metering
+    - q-vpn
+    - g-api
+    - g-reg
+    - cinder
+    - c-vol
+    - c-bak
+    - c-api
+    - c-sch
+    - s-proxy
+    - s-object
+    - s-container
+    - s-account
+    - heat
+    - h-api
+    - h-api-cfn
+    - h-api-cw
+    - h-eng
+    - ceilometer-acentral
```

Comment:
Here's a list of services that can be disabled:
* ceilometer
* heat
* q-metering and q-vpn

I think swift and c-bak can also be skipped.